com.unity.netcode.gameobjects/Runtime/SceneManagement/SceneEventData.cs

```diff
@@ -15,9 +15,6 @@ public class SceneEventData : IDisposable
     {
         /// <summary>
         /// The different types of scene events communicated between a server and client.
-        /// Scene event types can be:
-        /// A Server To Client Event (S2C)
-        /// A Client to Server Event (C2S)
         /// </summary>
         public enum SceneEventTypes : byte
```

Comment:
While you're renaming things, typically Enum type names are singular (so `SceneEventType`)